api.py

```
@@ -120,10 +120,11 @@ async def api_remove_swimmer(
 async def api_sync_swimmers(
     request: Request,
     db: Session = Depends(get_db),
+    scraper: SwimrankingsScraper = Depends(get_scraper),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)